tests/test_observers/test_gcs_observer.py

```diff
@@ -1,14 +1,12 @@
-#!/usr/bin/env python
-# coding=utf-8
 import pytest
 import datetime
 import os
 import json
 
-import hashlib
-import tempfile
 from sacred.observers import GoogleCloudStorageObserver
 
+from tests.conftest import tmpfile  # noqa F401
```

Comment:
A fixture from conftest.py should not need to be imported

Reply:
Thanks, I'm not too familiar with pytest internals.